packages/reports/addon/components/navi-date-picker.js

```diff
@@ -204,7 +205,7 @@ export default Ember.Component.extend({
 
       // Convert date to start of time period
       let dateTimePeriod = getIsoDateTimePeriod(get(this, 'dateTimePeriod'));
-      this.sendAction('dateSelected', moment(newDate).startOf(dateTimePeriod));
+      this.sendAction('onUpdate', moment(newDate).startOf(dateTimePeriod));
```

Comment:
Can we do:
 
```js
this.onUpdate( moment(newDate).startOf(dateTimePeriod));
```

Reply:
Tried but it looks like we'll need to do some refactoring to get rid of all the `sendAction` calls. I opened an issue #283